Jenkinsfile

```diff
@@ -201,792 +98,78 @@ pipeline {
        }
        stage('Unit Tests') {
            parallel {
-                stage('1 Ubuntu 18.04') {
+                stage('Ubuntu 20.04') {
                     agent {
-                        label "bionic-agent"
+                        label "ubuntu2004-agent"
                     }
                     steps {
                         catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
-                            unstash "bionic-unit-tests"
-                            dir ('test') {
-                                sh "./carta_backend_tests --gtest_output=xml:ubuntu_bionic_test_detail.xml"
+                            dir ('build/test') {
+                                sh "echo 'leak:*casacore*' >> ../../debug/asan/myasan-leaks.supp && echo 'leak:*casa*' >> ../../debug/asan/myasan-leaks.supp && echo 'leak:*libcurl*' >> ../../debug/asan/myasan-leaks.supp && echo 'leak:*libwcs*' >> ../../debug/asan/myasan-leaks.supp"
+                                sh "ASAN_OPTIONS=suppressions=../../debug/asan/myasan.supp LSAN_OPTIONS=suppressions=../../debug/asan/myasan-leaks.supp ASAN_SYMBOLIZER_PATH=llvm-symbolizer LSAN_OPTIONS=detect_leaks=0 ./carta_backend_tests --gtest_output=xml:ubuntu_test_detail.xml --gtest_filter=-ImageExprTest.ImageExprFails"
```

Comment:
I think that for better readability it should be:
```
LSAN_OPTIONS=suppressions=../../debug/asan/myasan-leaks.supp:detect_leaks=0
```
but if it is working that way...

Reply:
also, are you missing the environment variable `${WORKSPACE}`?

Reply:
It seems we can use `../../` instead of ${WORKSPACE} now. But maybe I'll go back to ${WORKSPACE} as it might make it easier to see the real path in the log files if we need to track down some problems.